werror.go

```diff
@@ -232,90 +237,113 @@ func (e *werror) Cause() error {
 	return e.cause
 }
 
+// stack returns the Stacktracer for this error or nil if there is none.
+func (e *werror) StackTrace() StackTrace {
+	return e.stack
+}
+
+// Message returns the message string for this error or nil if there is none.
+func (e *werror) Message() string {
+	return e.message
+}
+
 func (e *werror) SafeParams() map[string]interface{} {
-	safe, _ := ParamsFromError(e)
```

Comment:
Because we no longer special-case `*werror`s in `ParamsFromError`, we implement this here.